### ISSUE TITLE
CCI exception on Windows 10 masked by second error

### ISSUE THREAD

**Issue by acrosman** [SFDO-Tooling/CumulusCI#2092]
During the Snowfakery-related `generate_and_load_from_yaml ` task, on Windows 10 if there is an error triggered by the recipe CCI masks the error message by throwing a second error related to cleaning up the temp data that was built for the process.

**Reproduction steps**
1. Create a recipe that is mostly correct, but cannot successfully upsert into the org (in my case random text into a lookup field).
2. Run the generate_and_load_from_yaml with that recipe.
3. You will eventually get `Error: [WinError 267] The directory name is invalid: [path to temp.db]`

**Context**
Please include the following information at minimum.

Windows 10
```
cci version
CumulusCI version: 3.20.1 (C:\Users\ahcrosman\.local\bin\cci)
Python version: 3.8.6 (c:\users\ahcrosman\.local\pipx\venvs\cumulusci\scripts\python.exe)
```
Installed via pipx
Reproduceable in vscode powershell and powershell directly.

**Additional information**
Demoed for @prescod 10/14/20

Workaround:  add a working directory explicitly to the command.  `-o working_directory c:\mytempdir`

**Comment by prescod** [comment]
@acrosman : If you have a chance to repo and attach a "cci error info" or "cci error gist" that would be helpful. Otherwise this is a very detailed report. Thanks.

**Comment by prescod** [comment]
I believe that this is fixed in 3.24.0 released yesterday.

**Comment by acrosman** [comment]
For what it's worth, confirmed this is resolved for setup I was working from originally.
